upi/azure/deploy_azarm.sh

```diff
@@ -0,0 +1,17 @@
+echo "Start Deployment"
+az group deployment create \
+   --name $1 \
+   --resource-group $1 \
+   --template-uri "https://raw.githubusercontent.com/openshift/installer/master/upi/azure/azuredeploy.json" \
```

Comment:
it's better if this is from the local file compared to url.

Reply:
Done.